EXERCISE-1.md

```diff
@@ -198,7 +198,7 @@ this tutorial repo (i.e., `ngsdn-tutorial/tmp`). Example of these files are:
 
 ## 4. Program leaf1 using P4Runtime
 
-For this part we will use the [P4Runtime Shell](p4runtime-sh), an interactive
+For this part we will use the [P4Runtime Shell](util/p4rt-sh), an interactive
```

Comment:
This was meant to be a reference to the URL at the bottom of the file. It wasn't working because of the round parentheses instead of square ones. 

```suggestion
For this part we will use the [P4Runtime Shell][p4runtime-sh], an interactive
```

Reply:
I originally use p4c-graph but it seems have a bug to parse p4 code so I manually generate the graph for this pipeline.